components/vf-analytics-google/vf-analytics-google.js

```diff
@@ -189,6 +209,7 @@ function vfGaInit(vfGaTrackOptions) {
   // standard google analytics bootstrap
   if (vfGaTrackOptions.vfGaTrackPageLoad) {
     ga("send", "pageview");
```

Comment:
Here `ga` may not be defined if one moves to ga4. I think we may need to use the same checks for `gtag`